cmd/podman/pods/create.go

```diff
@@ -290,13 +290,3 @@ func replacePod(name string) error {
 	}
 	return removePods([]string{name}, rmOptions, false)
 }
-
-func MapOptions() {
```

Comment:
It looks like this has moved to common.  I'd like to suggest a name change to make it less generic there.  Perhaps PodsMapOptions?